Project_Ideas/README.md

```diff
@@ -16,3 +16,4 @@
 | Genuine Charity Applications | In these recent times, people are really interested in helping and giving back to the community, but they feel that will this help really reach the needy people? So in such cases, we can make a transparent and genuine charity application that will prove helpful to the community.The needy people would upload their data on the portal connected to the Blockchain system .The community can access their data from the Blockchain and would be able to help them directly and safely.| Not done||
 | Document storage systems |There are a lot of documents and licenses which are really very important for us, and we can be needing them any time. So we can build a secure platform with the help of blockchain which can ensure that our documents are safe and easy to access.The user will upload their documents to the Blockchain system.The government or any system which requires submission of documents can access them from the Blockchain with the required permissions from the user.This system if implemented would lead to secure, paperless, and quick transactions and would be convenient to the users as well as the government.| Not done||
 | Crowdfunding | Blockchain-powered crowdfunding is anonymous, safe, and borderless, it has the potential to be more democratic. Blockchain can help to speed up the development of a worldwide crowdfunding platform that anybody in the world may use. It will enable all-encompassing crowdfunding. Instead of giving money to a comapny or a third party , with the help of smart contract the people will be able to directly give money to the people who are asking for funds.
+| DAO | Decentralized Autonomous Organizations they are an evolution of organizations the way we know.People who invest in DAOs receive tokens and have the right to vote on DAO decisions because there is no centralized leadership. It is allows greater transparency, as all actions are recorded on an open code public record, allowing people to examine the history of actions.They can finance projects, communities, provide resources to create a pool of investments, a freelancer network, a charity fund , and so on. 

```

Comment:
DAO is not a project its a  term used in the blockchain . like voting application  comes in dao etc